src/constants/sinsemilla.rs

```diff
@@ -197,6 +222,22 @@ mod tests {
         );
     }
 
+    #[test]
+    fn q_note_zsa_commitment_m() {
+        let domain = CommitDomain::new(NOTE_ZSA_COMMITMENT_PERSONALIZATION);
```

Comment:
_From str4d ([r2542967008](https://github.com/zcash/orchard/pull/471#discussion_r2542967008))_

This should use `CommitDomain::new_with_separate_domains` to ensure it exactly matches the `CommitDomain` used in production. The fact that only the hash domain influences `Q` should not be something that the constant-checking test relies on.
```suggestion
        let domain = CommitDomain::new_with_separate_domains(
            NOTE_ZSA_COMMITMENT_PERSONALIZATION,
            NOTE_COMMITMENT_PERSONALIZATION,
        );
```

Reply:
FIXED